bpf/lib/nodeport.h

```diff
@@ -58,25 +58,6 @@ bpf_skip_nodeport(struct __ctx_buff *ctx)
 }
 
 #ifdef ENABLE_NODEPORT
-#ifdef ENABLE_IPV4
-struct {
-	__uint(type, BPF_MAP_TYPE_LRU_HASH);
-	__type(key, __be32);	/* ipv4 addr */
-	__type(value, union macaddr);	/* hw addr */
-	__uint(pinning, LIBBPF_PIN_BY_NAME);
-	__uint(max_entries, NODEPORT_NEIGH4_SIZE);
-} NODEPORT_NEIGH4 __section_maps_btf;
-#endif /* ENABLE_IPV4 */
-
-#ifdef ENABLE_IPV6
```

Comment:
I guess it doesn't matter much as it is just a type definition, but do we still need the `#ifdef ENABLE_IPV6` here (along with the corresponding `#endif`) as we presumably only use `dsr_opt_v6` when IPv6 is enabled?

Reply:
Removed it since it doesn't matter for struct definitions if not used.